pulpcore/app/serializers/content.py

```diff
@@ -18,12 +18,23 @@ class Meta:
         model = models.Content
         fields = base.MasterModelSerializer.Meta.fields
 
+    def create(self, validated_data):
+        _artifacts = validated_data.pop('_artifacts')
```

Comment:
I think this method needs to check for three cases: the existence of _artifacts, _artifact, or no _artifact/_artifacts.

Reply:
```suggestion
        if '_artifacts' in validated_data:
            _artifacts = validated_data.pop('_artifacts')
        elif '_artifact' in validated_data:
            _artifacts = [validated_data.pop('_artifact')]
        else:
            _artifacts = []
```

Reply:
Yeah, figured that out the hard way. I WIPed the PR accordingly. ;)

Reply:
Actually (i was discussing this with @asmacdo), we thought, the serializers validate should leave all validated_data in a common state. Another option is to move this create to the more specialized serializers.